docs/addons.md

```diff
@@ -69,7 +69,20 @@ spec:
     defaultIssuer: yourDefaultIssuer
 ```
 
-**Warning: cert-manager only supports one installation per cluster. If you are already running cert-manager, you need to remove this installation prior to enabling this addon. As long as you are using v1 versions of the cert-manager resources, it is safe to remove existing installs and replace it with this addon**
+**Warning: cert-manager only supports one installation per cluster. If you are already running cert-manager, you need to
+either remove this installation prior to enabling this addon, or mark cert-manger as not being managed by kOps (see below).
+As long as you are using v1 versions of the cert-manager resources, it is safe to remove existing installs and replace it with this addon**
+
+The following cert-manager configuration allows provisioning cert-manager externally and allows all dependent plugins
```

Comment:
Can you add the version macro?
```{{ kops_feature_table(kops_added_default='1.21', k8s_min='1.16') }}```

Reply:
I was not sure where to add this, I hope I got it right

Reply:
Usually we just add a subheading and place the macro under.

Reply:
Added another commit. I'm still unsure if this is how you expect it, if not, can you show me how you expect it?

Reply:
Looks perfect